modules/ejbca-ejb/src/org/ejbca/core/protocol/cmp/RevocationMessageHandler.java

```diff
@@ -292,6 +292,10 @@ public ResponseMessage handleMessage(final BaseCmpMessage msg, boolean authentic
 			}
 		} else if(StringUtils.equals(responseProtection, "signature")) {
 		    try {
+			    final X509Certificate cacert = (X509Certificate)ca.getCACertificate();
```

Comment:
Same here, just a reference to the RFC, and maybe a note that the recipient as sent by the sender may be the CA DN, but not byte-for-byte equal.

Reply:
done